docs/finch-daemon-config.md

```diff
@@ -16,6 +16,61 @@ Example usage:
 finch-daemon --socket-addr /tmp/finch.sock --debug --socket-owner 1001 --config-file /path/to/config.toml
 ```
 
+# Configuring finch-daemon to support socket activatione
```

Comment:
Can we just have an overall section for "Using finch-daemon with systemd", and also include the changes necessary to use socket activation?

I'm asking because we already have an example systemd service file in the README and I think it might be less confusing if we just link to this section instead of having multiple example systemd service files.